server/metrix/metric.py

```diff
@@ -80,6 +80,18 @@ def derivative_wrt_time(function, values, timestamps):
             actual_values = next_values
         return derivatives
 
+    @staticmethod
+    def calculate_length(function, values):
```

Comment:
tento nazov `calculate_length` sa mi moc nepozdava - neviem ci je to uplne to, co to robi, kedze ono to priamo nerata dlzku ale len to aplikuje danu funkciu na vsetky po sebe iduce dvojize v tom vstupnom zozname.

Ono sa mi to skor podoba na map...

A da sa to asi spravit aj kusok krajsie... https://www.geeksforgeeks.org/python-generate-successive-element-difference-list/ co povies?